storage/txcache/txCache.go

```diff
@@ -114,12 +123,89 @@ func (cache *TxCache) CountTx() int64 {
 	return cache.txByHash.counter.Get()
 }
 
+// Len is an alias for CountTx
+func (cache *TxCache) Len() int {
+	return int(cache.CountTx())
+}
+
 // CountSenders gets the number of senders in the cache
 func (cache *TxCache) CountSenders() int64 {
 	return cache.txListBySender.counter.Get()
 }
 
-// forEachSender iterates over the senders
+// forEachSender iterates over the senders in the cache
 func (cache *TxCache) forEachSender(function ForEachSender) {
 	cache.txListBySender.forEach(function)
 }
+
+// ForEachTransaction iterates over the transactions in the cache
+func (cache *TxCache) ForEachTransaction(function ForEachTransaction) {
+	cache.txByHash.forEach(function)
+}
+
+// Clear clears the cache
+func (cache *TxCache) Clear() {
+	cache.txListBySender.clear()
+	cache.txByHash.clear()
+}
+
+// Put is not implemented
+func (cache *TxCache) Put(key []byte, value interface{}) (evicted bool) {
+	return false
```

Comment:
These are easy to implement, I do not see why are these not implemented. It will be hard to follow when somebody uses the interface and a function does not do what it tells.

Reply:
Any implementation of these is dead code. We notify the programmer that they are not actually implemented. This workaround was applied in order to keep both old and new implementation - the coupling to `storage.Cacher` (a quite large interface) lead to this workaround.

Is it ok if we leave them as they are? That is, not implemented, but with logging.